it/lessons/advanced/concurrency.md

```diff
@@ -0,0 +1,175 @@
+---
+layout: page
+title: Concorrenza
+category: advanced
+order: 4
+lang: it
+---
+
+Uno dei punti di forza di Elixir è il suo supporto alla concorrenza. Grazie alla ErlangVM, scrivere programmi concorrenti in Elixir è più semplice di quanto uno possa aspettarsi. Il modello di concorreza si basa sugli Attori, processi indipendenti che comunicano con altri processi, attraverso il passaggio di messaggi.
+
+In questa lezione daremo un'occhiata ai moduli sulla concorrenza che vengono provveduti con l'installazione di Elixir. Nel prossimo capitolo, ci occupiamo dei _behaviour OTP_ che ci permettono di scrivere programmi concorrenti.
+
+## Tavola dei Contenuti
+
+- [Processi](#processi)
+  - [Message Passing](#message-passing)
+  - [Process Linking](#process-linking)
+  - [Process Monitoring](#process-monitoring)
+- [Agenti](#agenti)
+- [Task](#task)
+
+## Processi
+
+I processi che vivono nella ErlangVM sono leggeri, e possono tenere occupate tutte le CPU del sistema. Nonostante questi possano sembrare thread nativi, i processi Erlang sono più semplici, e non è raro trovare migliaia di processi concorrenti in un'applicazione Elixir.
+
+Il modo più semplice per lanciare un processo è attraverso il comando `spawn`, che accetta una funzione anonima o pre-definita. Quando creiamo un processo, il suo _Process Identifier_, o PID, ci viene restituito; con esso, possiamo identificare unicamente il processo creato all'interno della nostra applicazione.
```

Comment:
I see @apeacox is using "funzione con un nome" instead of "funzione predefinita"(link: https://github.com/apeacox/elixir-school/blob/master/it/lessons/basics/functions.md#funzioni-con-un-nome). The first sounds more correct while the second sounds much better. Maybe we should introduce a glossary, I see the russian translators introduced one in the root dir, we should keep one within the /it folder.